scripts/exgdas_global_marine_analysis_ecen.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+# exgdas_global_marine_analysis_ecen.py
+# This script creates an MarineRecenter class
+# and runs the initialize, run, and finalize  methods
+# which currently are stubs
+import os
+
+from wxflow import Logger, cast_strdict_as_dtypedict
+from soca.marine_recenter import MarineRecenter
```

Comment:
Please add a comment with a `TODO:` to move `marine_recenter.py` to `ush/python/pygfs/task` in the global-workflow so that when the time comes, it is easily achieved.

Reply:
If it helps, it would be:
```python3
from pygfs.task.marine_recenter import MarineRecenter
```